lib/matplotlib/axes/_base.py

```diff
@@ -566,11 +566,26 @@ def __init__(self, fig, rect,
         if self.yaxis is not None:
             self._ycid = self.yaxis.callbacks.connect('units finalize',
                                                       self.relim)
-        self.tick_params(top=rcParams['xtick.top'],
-                         bottom=rcParams['xtick.bottom'],
-                         left=rcParams['ytick.left'],
-                         right=rcParams['ytick.right'],
-                         which='both')
+
+        self.tick_params(top=rcParams['xtick.top'] and
+                         rcParams['xtick.minor.top'],
```

Comment:
I agree that is a bit confusing, however they both don't fit on the same line within 79 characters.